common/src/main/resources/assets/wynntils/lang/en_us.json

```diff
@@ -857,13 +857,13 @@
   "feature.wynntils.mainMap.description": "Adds a map to view the world.",
   "feature.wynntils.mainMap.holdGuildMapOpen.description": "Should Control need to be held to open guild territories on the map. If this is disabled, control will act as a toggle.",
   "feature.wynntils.mainMap.holdGuildMapOpen.name": "Hold Control to Show Guild Territories",
+  "feature.wynntils.mainMap.mapFeatureScale.description": "How big should map map features be?",
+  "feature.wynntils.mainMap.mapFeatureScale.name": "Map Point of Interest Scale",
```

Comment:
I think we should drop the "Point of Interest" name and just go with "map feature" (or just "feature", if the context is clear).

Reply:
Yes, my mistake, I wanted to do this originally.